code/modules/mob/living/simple_animal/hostile/mining_mobs/necropolis_tendril.dm

```diff
@@ -26,6 +26,7 @@
 	del_on_death = 1
 	var/gps = null
 	var/obj/effect/light_emitter/tendril/emitted_light
+	layer = TABLE_LAYER
```

Comment:
by the time the chest spawns, this mob is gone.

Reply:
Fuck, Gimme a bit I'll track down the one I need to then.